client/App.jsx

```diff
@@ -13,23 +13,61 @@ class App extends Component {
       rooms: 0,
       adults: 0,
       children: 0,
+      lowest: '',
+      secondLowest: '',
+      thirdLowest: '',
     };
+    this.getLowestPrices = this.getLowestPrices.bind(this);
+    this.updatePrices = this.updatePrices.bind(this);
   }
 
   componentDidMount() {
-    console.log('current state date type:', typeof this.state.checkInDate)
+    this.getLowestPrices();
+  }
+
+  getLowestPrices() {
     fetch(`http://localhost:8080/prices/${this.state.checkInDate}`)
-      .then(res => res.text())
-      .then(data => console.log(data));
+      .then(res => res.json())
+      .then((data) => {
+        const tupleArray = Object.entries(data.result[0]);
+        const sortedArray = tupleArray.sort((a, b) => {
+          return a[1] > b[1] ? 1 : -1;
+        });
+        console.log(sortedArray);
+        this.setState({
+          lowest: sortedArray[1],
+          secondLowest: sortedArray[2],
+          thirdLowest: sortedArray[3],
+        });
+      });
+  }
+  
+  updatePrices(state) {
+    this.setState({
+      lowest: state.lowest,
+      secondLowest: state.secondLowest,
+      thirdLowest: state.thirdLowest,
+    });
   }
 
   render() {
     return (
       <div>
         <span>Lowest prices for your stay</span>
         <span>
-          <CheckInDate checkInDate={this.state.checkInDate} />
-          <CheckOutDate checkOutDate={this.state.checkOutDate} />
+          <CheckInDate
+            lowest={this.state.lowest}
+            secondLowest={this.state.secondLowest}

```

Comment:
Destructure this.state in render method (prior to return statement) for more concise code